hack/make-rules/verify.sh

```diff
@@ -220,9 +228,13 @@ if ${QUICK} ; then
   echo "Running in quick mode (QUICK=true). Only fast checks will run."
 fi
 
+shopt -s globstar
+export API_KNOWN_VIOLATIONS_DIR="${KUBE_ROOT}"/api/api-rules
 ret=0
-run-checks "${KUBE_ROOT}/hack/verify-*.sh" bash
-run-checks "${KUBE_ROOT}/hack/verify-*.py" python3
+for base in ./hack staging/**/hack; do
```

Comment:
Finding "hack" anywhere feels sketchy.  Could we only do it in the root of modules?

Reply:
Done!